test/OpenTelemetry.Exporter.Geneva.Tests/UnixUserEventsDataTransportTests.cs

```diff
@@ -246,231 +245,6 @@ private static void EnsureUserEventsEnabled()
             throw new NotSupportedException("Kernel does not support user_events. Verify your distribution/kernel supports user_events: https://docs.kernel.org/trace/user_events.html.");
         }
     }
-
-    private sealed class ConsoleCommand : IDisposable
```

Comment:
Check if it can be removed and created as separate PR. As it a refactoring easy to review and merge.

Reply:
Split into PR #2654.